Extensions/pokes.js

```diff
@@ -25,6 +26,32 @@ XKit.extensions.pokes = {
 		XKit.extensions.pokes.checkEligibility();
 	},
 
+	fetch_pokedex: function(callback, error) {
+		if (!XKit.extensions.pokes.gist_cache) {
+			GM_xmlhttpRequest({
+				method: "GET",
+				url: XKit.extensions.pokes.pokedex_url,
+				json: true,
+				onerror: error || function(response) {
```

Comment:
not sure i'm a super huge fan of this formulation. Mind just doing the same thing we do in the catch? Its more explicit/less "clever".